Documentation/Signoffs/WorkSysten(1).md

```diff
@@ -0,0 +1,161 @@
+
+
+### Meeting Magnetic Sensor Range Constraint
+
+The Hall effect sensor has a voltage draw of 3.3 V to 5 V that produces a magnetic flux density of 39 mT, equivalent to 390 Gauss [1]. Based on these specifications there must be a magnet opposite of the sensor capable of producing a magnetic flux density of at least 390 Gauss to trigger the sensor. By using a 1 ¼” diameter by ¼” thickness nickel-based magnet it will produce 456 Gauss at a distance of 22 mm. The distance of 22 mm is found to be the largest possible distance between the magnet and the sensor. The space between the bike spoke and the frame is 30 mm, minus the thickness of the magnet (6.35 mm) and the thickness of the sensor (2.06 mm) which equals 21.59 mm which is then rounded to 22 mm to ensure that it will reach a slightly above 21 mm.
+
+- The residual induction for the magnet chosen is 14,800 gauss. The thickness of the magnet is 0.25 inches or 6.35 millimeters. The radius of the magnet is 0.625 inches or 15.875 millimeters. The maximum distance from the magnet and sensor we calculated was rounded to 22 mm. Using the following formulas we can solve for gauss at maximum distance to prove that it is above the 390 gauss required by the sensor.
+**1 Tesla = 10,000 Gauss**
+
+
+  ![Magnetic Flux Calculation](https://github.com/Baebel43/team5capstone/blob/main/Documentation/Images/Magentic%20Flux%20Calculation%20Equation.png)
+
+- Plugging in all the values: Br is 14800, D is 6.35, R is 15.875, and z is 22 mm we get  455.8 gauss which is well above the required value for the sensor.
+
+The Hall effect sensor has a quiescent voltage between 0.535 V and 0.665 V [1]. This is used for the range of the linear output voltage which has a minimum of the quiescent voltage and a maximum of the voltage delivered to the device (3.3 V or 5 V) minus 0.2 V [1]. Using an Arduino Nano 33 BLE we can set the voltage threshold to the driving voltage of the Hall effect sensor minus 0.2 V to be a 1 and anything below this value to be a 0.
+
+### Meeting the Diameter of Wheel Constraint
+
+The minimum wheel size has been set at 18 inches to account for the average wheel size of the smallest type of bike that would be commonly ridden on a trail, a BMX bike [3]. The largest wheel size has been set at 32 inches which is 3 inches above the most common largest mountain bike tire at 29 inches [4]. Additionally, this constraint for minimum wheel size is set to ensure that the device should not be fully submerged in water. If the rider were to hit a 9-inch deep puddle it would most likely not only result in the destruction of the device but also a failure of the bicycle.
+
+- This is proven within the calculations for velocity
+
+
+### Meeting the Sensing Speed Constraint

```

Comment:
I don't understand how you are proving this. This needs to be more explicitly addressed. What is the nyquist rate for your application? How fast can you system sample? These have to be addressed clearly.

Reply:
Show the math and give comment where necessary. Don't talk about it in the abstract without actually doing the math.